back-end/objects/RModelWrapper.py

```diff
@@ -52,6 +52,41 @@ def __init__(
         else:
             print("Checkpoint file not found: {}".format(net_path))
 
+        # TODO: Should remove this in the future. Currently added for passing the e2e test.
+        # Also stop passing app to this class.
+        self.upload_model_4_e2e_test(app)
+
+    def upload_model_4_e2e_test(self, app):
+        import io
+        import contextlib
+
+        metadata_4_save = {
+            "class_name": self.modelwork_type,
+            "nickname": "simple-classifier",
+            "predefined": True,
+            "pretrained": False,
+            "description": "This is the model for testing.",
+            "tags": ["test"],
+            "create_time": datetime.now(),
+            "code_path": None,
```

Comment:
Just fyi, I think you can simply ignore `None` fields (and maybe `0` fields as well). They will default to null in database.

Reply:
Got it